.github/ISSUE_TEMPLATE/improve-modules.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Improvement of module
+about: Improve the content of existing module.
+title: ''
+labels: Enhance
+assignees: ''
+---
+
+## Which module you would like to maintain / improve?
```

Comment:
I'm not sure if `maintain` should be kept here. However, I'm also not sure if it should become its own template.

Reply:
after some thoughts, I feel if someone wants a complete rework of the module, they should submit as a new module.

for maintainance, we are mostly looking for people to check the links are not dead etc